qiskit/synthesis/clifford/clifford_decompose_layers.py

```diff
@@ -389,3 +399,31 @@ def _check_gates(qc, allowed_gates):
     for inst, _, _ in qc.data:
         if not inst.name in allowed_gates:
             raise QiskitError("The gate name is not in the allowed_gates list.")
+
+
+def synth_clifford_depth_lnn(cliff):
```

Comment:
Maybe change the name to something like `synth_clifford_layers_lnn`?

Reply:
I prefer `synth_clifford_depth_lnn` since optimized depth algorithms for LNN (like Maslov-Yang) will destroy some of the layered structure.